plone/app/locales/locales/de/LC_MESSAGES/widgets.po

```diff
@@ -574,7 +574,7 @@ msgstr "Aktualisieren"
 
 #: ./patterns/resourceregistry/pattern.js
 msgid "Registry"
-msgstr ""
+msgstr "Registrierung"
```

Comment:
The Registry is the name of the component to configure ressources (i.e. the first tab in http://demo.plone.de/@@resourceregistry-controlpanel), not to regierster Users. So I'd either use the default or use "Ressourcen"